Arrowgene.Ddon.GameServer/Handler/SkillGetAbilityCostHandler.cs

```diff
@@ -18,8 +18,8 @@ public override void Handle(GameClient client, StructurePacket<C2SSkillGetAbilit
         {
             client.Send(new S2CSkillGetAbilityCostRes()
             {
-                CostMax = 100
+                CostMax = 100U + client.Character.ExtendedParams.AbilityCost
```

Comment:
Shouldn't it be without adding 100?

Reply:
Before I added this, the start menu was showing 0/0. Is it safe to assume the baseline is always 100 points + whatever was gained from the dragon?

Reply:
The baseline wasn't 100, I don't remember the exact, value, you may have to find it in old footage

Reply:
So should we set this to 15 as the base?

Reply:
Seeing thats what it was in the OG server I'd say yes